tests/mocks.py

```diff
@@ -733,6 +733,27 @@ def comment(self, content, author=None):
     def conduit_ping(self):
         return "ip-123-123-123-123.us-west-2.compute.internal"
 
+    @conduit_method("bugzilla.account.search")
+    def bugzilla_account_search(self, phids=None, ids=None):
+        """Return a list of Bugzilla IDs and Phabricator phids, given phids or ids."""
+        if ids:
+            raise NotImplementedError("Searching by Bugzilla ID is not implemented.")
```

Comment:
Not implemented in the test mock, or not implemented in the Conduit API endpoint?

Reply:
> Not implemented in the test mock, or not implemented in the Conduit API endpoint?

In the test mock -- I can be more clear about that.